object-store-client-play-26/src/test/scala/uk/gov/hmrc/objectstore/client/play/PlayObjectStoreClientEitherSpec.scala

```diff
@@ -32,6 +32,7 @@ import play.api.inject.bind
 import play.api.inject.guice.GuiceApplicationBuilder
 import play.api.libs.functional.syntax._
 import play.api.libs.json._
+import uk.gov.hmrc.objectstore.client.ObjectExpirationPeriod.one_week
```

Comment:
just importing `ObjectExpirationPeriod` gives a nice namespace when using the value